extensions/objects/models/objects.py

```diff
@@ -364,26 +364,18 @@ class ListOfTabs(BaseObject):
     description = 'Tab content that contains list of tabs.'
 
     SCHEMA = {
-        "type": "list",
-        "items": {
-            "type": "dict",
-            "properties": [{
-                "name": "title",
-                "description": "Tab title",
-                "schema": {
-                    "type": "unicode",
-                    "validators": [{
```

Comment:
Why remove the validators? Ditto below.

Reply:
I think that's conceptually incorrect. If this is a ListOfTabs object it needs to have the full schema (including validators) that applies to that object. There are other validators in this file too for other objects so I don't think validators is out of scope for this file.

Basically, ignore rich_text_component_definitions.js when modifying this file. This file should be conceptually self-contained.

Reply:
Yes, got it.
Done, PTAL!